exercises/concept/squeaky-clean/.meta/design.md

```diff
@@ -0,0 +1,36 @@
+# Design
+
+## Learning objectives
+
+- Know of the existence of the `char` type.
+- Know what a char represents (a Unicode character).
+- Know how to define a `char`.
+- Know that a `char` is not the same as a single character string
+- Know how to access a `char` in a string by index.
+- Know of some basic `char` methods (like converting to uppercase).
+- Know that `char`s are immutable.
```

Comment:
Do we talk about this? It might not be necessary in Elm, where nothing is mutable